src/Optimization.js

```diff
@@ -24,4 +25,38 @@ module.exports = class extends ChainedMap {
       'usedExports',
     ]);
   }
+
+  minimizer(name) {
+    if (!this.minimizers.has(name)) {
+      this.minimizers.set(name, new Plugin(this, name));
+    }
+
+    return this.minimizers.get(name);
+  }
+
+  toConfig() {
+    return this.clean(
+      Object.assign(this.entries() || {}, {
+        minimizer: this.minimizers.values().map(plugin => plugin.toConfig()),
+      })
+    );
+  }
+
+  merge(obj, omit = []) {
+    const omissions = [];
```

Comment:
Let's remove `omission` since there aren't any at the moment (looking at other implementations of `merge()`, the others don't include it if not used)